packages/core/src/rum/instrumentation/resourceTracking/distributedTracing/distributedTracingHeaders.ts

```diff
@@ -221,16 +238,22 @@ const generateTraceContextHeader = ({
 
 const generateTraceStateHeader = ({
     parentId,
-    isSampled
+    isSampled, 
+    rumSessionId
 }: {
     parentId: SpanId;
     isSampled: boolean;
+    rumSessionId: string | null;
 }) => {
     const sampled = `s:${isSampled ? '1' : '0'}`;
     const origin = 'o:rum';
     const parent = `p:${parentId.toString(TracingIdFormat.paddedHex)}`;
-
-    return `dd=${sampled};${origin};${parent}`;
+    if (rumSessionId) {
+        const session = `t.rsid:${rumSessionId}`;
+        return `dd=${sampled};${origin};${parent};${session}`;
+    } else {
+        return `dd=${sampled};${origin};${parent}`;
+    }
```

Comment:
```suggestion
       const traceStateHeader = `dd=${sampled};${origin};${parent}`;

    if (rumSessionId) {
        const session = `t.rsid:${rumSessionId}`;
        return `${traceStateHeader};${session}`;
    }

    return traceStateHeader;
```

**MINOR:** Just a nitpick, perhaps we could turn the base `traceStateheader` into a const to make things a bit clearer. I'm not so sure about the name though.